browser/ui/webui/settings/brave_appearance_handler.cc

```diff
@@ -99,6 +104,15 @@ void BraveAppearanceHandler::GetIsSuperReferralActive(
                             base::Value(IsSuperReferralActive(profile_)));
 }
 
+void BraveAppearanceHandler::GetIsBinanceSupported(
```

Comment:
This should not call into `binance::IsBinanceSupported` and just resolve to false if `#if !BUILDFLAG(BINANCE_ENABLED`

Reply:
done